### ISSUE TITLE
Plunder Isles to Southsea Culture

### ISSUE THREAD

**Issue by zumbak04** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#552]
**Describe your suggestion in full detail below:**
Southsea culture is very small so it's on the verge of extinction. If we want to prevent it, we have to convert the majority of the Plunder Isles to Southsea. I suggest turning Gurubashi, Hozen lands to Southsea (since they have a strong presence in the world and region anyway.

**Comment by Remerod** [comment]
Could we not add some random Islands to the area between Plunder Isles and Pandaria?. i really like the plunder isles as it is, a bunch on different cultures all grasping for dominance, I personally think it is the most "Sandbox" start in the base mod.

I know we gotten some of the island expedition islands in right? but not all right? even if we have there most be some lore islands or wc3 islands that's not in the mod that we could but there?

<details>
<summary>Click to expand</summary>

Uncharted Islands
![image](https://user-images.githubusercontent.com/79544635/126903657-a1f5c511-3a0f-42b1-9199-5596bd6fa51c.png)

Some Unused but planned islands?
![image](https://user-images.githubusercontent.com/79544635/126903773-94ce617a-ae81-4c7c-b35b-4b66253122d9.png)
![image](https://user-images.githubusercontent.com/79544635/126903787-1888f5f5-70c4-4060-ad76-240c8ac6263f.png)

Wc3 islands not placed on the map might might fit the region
Mystic Isles
![image](https://user-images.githubusercontent.com/79544635/126904157-b85af639-931b-4051-b3db-2390b0ce0da0.png)
Petrified forest
![image](https://user-images.githubusercontent.com/79544635/126904223-5332cd01-1a34-49d8-bb97-4e04f66d805b.png)
Emerald Shores
![image](https://user-images.githubusercontent.com/79544635/126904432-3caccea2-58dd-44a6-ae0d-5a2cf1f9e2f6.png)
Sunrock Cove
![image](https://user-images.githubusercontent.com/79544635/126904470-b262eb62-4473-42cd-bf83-13752b02cb2f.png)
Typhoon
![image](https://user-images.githubusercontent.com/79544635/126904580-51f4b387-7b5a-4341-befc-cb9f590542ee.png)
Hurricane Isle
![image](https://user-images.githubusercontent.com/79544635/126904633-4879532d-407e-410c-ac87-919a26b9f9d0.png)
Mur'gul Oasis (could be part of an island)
![image](https://user-images.githubusercontent.com/79544635/126904680-0fbc4e82-0022-40a4-9821-3f854965daa6.png)
Spider Falls
![image](https://user-images.githubusercontent.com/79544635/126904729-723dfdc5-c972-44a9-9de7-b257eeb8b30d.png)
Twilight Ruins
![image](https://user-images.githubusercontent.com/79544635/126904757-bd2feca6-596d-4e7f-8a6e-33048aed8660.png)
Round Village (sound like it could be near pandaria and might have asian looking dudes on it, i dunno)
![image](https://user-images.githubusercontent.com/79544635/126904802-9d4a522b-0dfb-4b0f-a0b4-210c64d67bc0.png)
Emerald Gardens
![image](https://user-images.githubusercontent.com/79544635/126904834-d7d0d3a5-5fe7-41e0-bdd1-9177b52e3dc8.png)

Random Island possibility child paint
![image](https://user-images.githubusercontent.com/79544635/126904973-2c2eb8db-c2b0-4288-988e-ba69327c6d1f.png)

</details>

I'm sure there are others examples but I think some of these would fit for the **South Sea** culture and maybe others to making it similar to Plunder Isles, but then it might just be repeating the problem again.

**Comment by CantTouchThisJinTao** [comment]
Yeah personally I'd love a few more islands to be added, gives a few more interesting start positions where you can carve out your own kingdom.
I also like how Plunder isles are a nice mix of cultures, makes the region interesting.
Maybe if some of these other islands Remerod mentioned can fit in logically (not done any research into them myself yet) then they can be the new Plunder isles (mix of cultures etc) and Plunder isles could be mostly Southsea with like 1 or 2 Murloc culture regions?

**Comment by Loralius** [comment]
How about adding [Exile's Reach](https://wowpedia.fandom.com/wiki/Exile%27s_Reach), the new starter zone? It is controlled by Ogres by 9.0.1 with populations of murloc, harpy, tortollan and quillboar, but Vrykul settlements are also present on the outlying isles. Ogres did not exist on Azeroth by the 583 bookmark so its status at that time can certainly include some pirates, in addition to other cultures.
